build-utils/findmodifiedprojects.sh

```diff
@@ -3,8 +3,8 @@
 # Can run anywhere within the repo
 # If a file from a common area, like the root diretory was modified, we call that _SHARED_
 set +x
-#ignore these file name patterns when figuring out what modules changed 
-declare -a exclude_patterns=( '^.*\.md' '^.*.pdf' )
+#ignore these file name patterns when figuring out what modules changed
+declare -a exclude_patterns=( '^.*\.md' '^.*.pdf', '^.*\.spec\.ts' )
```

Comment:
Not sure why tests were not part of patterns to exclude, but it doesn't make sense to build and deploy studies to Dev env if only tests were changed in PR. All Playwright E2E tests are included in new pattern. From now on, merging Playwright test PRs will not trigger build and deploy studies to Dev env anymore.